helmfile.d/20-s3.yaml

```diff
@@ -50,21 +54,30 @@ releases:
     set:
       - name: cc.enabled
         value: {{ .Values.confluent_cloud.enabled }}
+      {{- if not .Values.radar-s3-connector.bucketAccessKey }}
```

Comment:
Shouldn’t these values and the others use underscores? `radar_s3_connector`

Reply:
Fixed in a9560880009df7590318966357bd1cf964ff3a43